src/test/scala/chiselTests/Counter.scala

```diff
@@ -57,6 +70,10 @@ class CounterSpec extends ChiselPropSpec {
     forAll(safeUInts) { (seed: Int) => whenever(seed >= 0) { assertTesterPasses{ new EnableTester(seed) } } }
   }
 
+  property("Counter can be reset") {
+    forAll(smallPosInts) { (seed: Int) => assertTesterPasses{ new ResetTester(seed) } }
+  }
+
```

Comment:
This test is probably overkill and it seems to repeat (at least locally, for me) seeds. Out of scope for this PR, though. I like having this PR consistent with the other tests here.